src/shared/propTypes.js

```diff
@@ -99,7 +99,7 @@ export const tileGroupProps = {
 };
 
 export const tileProps = {
-  activeStartDate: PropTypes.instanceOf(Date).isRequired,
```

Comment:
Please keep it as `isRequired` - `tileDisabled` should always get this parameter, so it *is* required.